pkg/machine/ignition/ignition.go

```diff
@@ -441,51 +358,27 @@ Delegate=memory pids cpu io
 			},
 		})
 	}
-
-	// Set delegate.conf so cpu,io subsystem is delegated to non-root users as well for cgroupv2
-	// by default
-	files = append(files, File{
-		Node: Node{
-			Group: GetNodeGrp("root"),
-			Path:  "/etc/systemd/system/user@.service.d/delegate.conf",
-			User:  GetNodeUsr("root"),
-		},
-		FileEmbedded1: FileEmbedded1{
-			Append: nil,
-			Contents: Resource{
-				Source: EncodeDataURLPtr(delegateConf),
-			},
-			Mode: IntToPtr(0644),
-		},
-	})
-
-	// Add a file into linger
-	files = append(files, File{
-		Node: Node{
-			Group: GetNodeGrp(usrName),
-			Path:  "/var/lib/systemd/linger/core",
-			User:  GetNodeUsr(usrName),
-		},
-		FileEmbedded1: FileEmbedded1{Mode: IntToPtr(0644)},
-	})
-
+	// TODO Commenting this out because I think we can but leaving as a
+	// breadcrumb until we are certain
 	// Set deprecated machine_enabled to true to indicate we're in a VM
-	files = append(files, File{
-		Node: Node{
-			Group: GetNodeGrp("root"),
-			Path:  "/etc/containers/containers.conf",
-			User:  GetNodeUsr("root"),
-		},
-		FileEmbedded1: FileEmbedded1{
-			Append: nil,
-			Contents: Resource{
-				Source: EncodeDataURLPtr(rootContainers),
-			},
-			Mode: IntToPtr(0644),
-		},
-	})
-
-	// Set machine marker file to indicate podman is in a qemu based machine
+	// files = append(files, File{
```

Comment:
this is safe to remove, I opened https://github.com/containers/common/pull/1867 to remove it completely from containers.conf